webapp/graphite/render/views.py

```diff
@@ -211,6 +213,7 @@ def renderView(request):
         add_never_cache_headers(response)
       log.rendering('JSON rendering time %6f' % (time() - jsonStart))
       log.rendering('Total request processing time %6f' % (time() - start))
+      log_expensive_request(start, request)
```

Comment:
it almost feels like the purpose of the render logger is to log query times. should we leverage that in combination with a `SLOW_QUERY_THRESHOLD` setting? it can default to 0 to behave as is today.